latch_cli/services/cp/remote_copy.py

```diff
@@ -0,0 +1,92 @@
+import click
+import gql
+from gql.transport.exceptions import TransportQueryError
+
+from latch.gql._execute import execute
+from latch_cli.services.cp.ldata_utils import LDataNodeType, get_node_data
+from latch_cli.services.cp.path_utils import get_name_from_path, get_path_error
+
+
+# todo(ayush): figure out how to do progress for this
```

Comment:
Since this is handled by ribosome, ribo would need to have some reporting. Likely a db subscription from here that listens to events for the particular copy request. This would also require threading through a copy_id or something.